scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java

```diff
@@ -104,18 +106,15 @@
 @NamedQueries({ @NamedQuery(name = "deleteTaskDataInBulk", query = "delete from TaskData where jobData.id in :jobIdList"),
                 @NamedQuery(name = "countTaskData", query = "select count (*) from TaskData"),
                 @NamedQuery(name = "countTaskDataNotFinished", query = "select count (*) from TaskData where taskStatus <> org.ow2.proactive.scheduler.common.task.TaskStatus.FINISHED"),
-                @NamedQuery(name = "getFinishedTasksCount", query = "select count(*) from TaskData task where taskStatus in (:taskStatus) and task.jobData.removedTime = -1"),
+                @NamedQuery(name = "getTasksCountByStatus", query = "select count(*) from TaskData task where taskStatus in (:taskStatus)"),
                 @NamedQuery(name = "getMeanTaskPendingTime", query = "select avg(startTime - :jobSubmittedTime) from TaskData task where task.jobData.id = :id and task.startTime > 0"),
                 @NamedQuery(name = "getMeanTaskRunningTime", query = "select avg(task.finishedTime - task.startTime) from TaskData task where task.startTime > 0 and task.finishedTime > 0 and task.jobData.id = :id"),
-                @NamedQuery(name = "getTasksCount", query = "select count(*) from TaskData task where taskStatus = :taskStatus and task.jobData.removedTime = -1"),
-                @NamedQuery(name = "getTasksCountForUsername", query = "select count(*) from TaskData task where task.jobData.owner = :username and taskStatus in (:taskStatus) and task.jobData.removedTime = -1"),
-                @NamedQuery(name = "getPendingTasksCount", query = "select count(*) from TaskData task where taskStatus in (:taskStatus) and task.jobData.status in (:jobStatus) and task.jobData.removedTime = -1"),
-                @NamedQuery(name = "getRunningTasksCount", query = "select count(*) from TaskData task where taskStatus in (:taskStatus) " +
-                                                                   "and task.jobData.status in (:jobStatus) and task.jobData.removedTime = -1"),
+                @NamedQuery(name = "getTasksCount", query = "select count(*) from TaskData task where taskStatus = :taskStatus"),
+                @NamedQuery(name = "getTasksCountForUsername", query = "select count(*) from TaskData task where task.jobData.owner = :username and taskStatus in (:taskStatus)"),
                 @NamedQuery(name = "findTaskData", query = "from TaskData where id in (:ids)"),
                 @NamedQuery(name = "findTaskDataById", query = "from TaskData td where td.id = :taskId"),
                 @NamedQuery(name = "getTotalNumberOfHostsUsed", query = "select count(distinct executionHostName) from TaskData task where task.jobData.id = :id"),
```

Comment:
@fviale so, u decided to not replace `task.jobData.id` by `task.id.jobId` to avoid the join ? There are 2 occurrences of  `task.jobData.id` to be potentially replaced.

Reply:
I agree and will do it